cmd/chart-repo/utils_test.go

```diff
@@ -73,6 +73,24 @@ func (h *goodHTTPClient) Do(req *http.Request) (*http.Response, error) {
 	return w.Result(), nil
 }
 
+type chartHTTPClient struct{}
```

Comment:
I guess we can remove this one and the `Do` below now that we have the emptyChartRepoHTTPClient?

Reply:
Looks like I forgot to remove this one. :) 
Removed this one now.